doc/reference/configuration/configuration_reference.rst

```diff
@@ -1249,6 +1217,149 @@ The ``database`` section defines database-specific configuration parameters, suc
     | Default: false
     | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
 
+..  _configuration_reference_fiber:
+
+fiber
+-----
+
+The ``fiber`` section describes options related to configuring :ref:`fibers, yields, and cooperative multitasking <concepts-coop_multitasking>`.
+
+..  NOTE::
+
+    ``fiber`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+* :ref:`fiber.io_collect_interval <configuration_reference_fiber_io_collect_interval>`
+* :ref:`fiber.too_long_threshold <configuration_reference_fiber_too_long_threshold>`
+* :ref:`fiber.worker_pool_threads <configuration_reference_fiber_worker_pool_threads>`
+* :ref:`fiber.slice.* <configuration_reference_fiber_slice_options>`
+* :ref:`fiber.top.* <configuration_reference_fiber_top_options>`
+
+..  _configuration_reference_fiber_io_collect_interval:
+
+..  confval:: fiber.io_collect_interval
+
+    The :ref:`fiber <app-fibers>` will sleep for the defined number of seconds
```

Comment:
If possible, I'd use the same pattern for all config option descriptions. Smth like:

_The time period (in seconds) a fiber sleeps between iterations of the event loop._